### PR TITLE
Phone number validation

### PR DESCRIPTION
**Pull request by zsbrown97**
Phone number validation better than ever. Zero duct tape used in the final product.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
```
Comment (HootDunk) [inline package.json:1]: This package json can be deleted

File: package.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
   "dependencies": {
-    "@types/geojson": "^7946.0.13",
-    "@types/leaflet": "^1.9.8",
-    "leaflet": "^1.9.4"
+    "@types/google-libphonenumber": "^7.4.30",
+    "google-libphonenumber": "^3.2.34"
```
Comment (HootDunk) [inline package.json:4]: move this to dependencies in `frontend/package.json`

File: package.json [hunk 3]
```diff
@@ -1,7 +1,9 @@
 {
   "dependencies": {
-    "@types/geojson": "^7946.0.13",
-    "@types/leaflet": "^1.9.8",
-    "leaflet": "^1.9.4"
+    "@types/google-libphonenumber": "^7.4.30",
```
Comment (HootDunk) [inline package.json:3]: move this to devDependencies in `frontend/package.json`

File: package.json [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
+    "google-libphonenumber": "^3.2.34"
+  },
+  "devDependencies": {
+    "@types/node": "^20.11.16"
```
Comment (HootDunk) [inline package.json:7]: Do we need this dependency?

File: frontend/package.json
Comment (JackDiSalvatore) [inline frontend/package.json:1]: I think this is good, but @HootDunk should `google-libphonenumber` be a non-dev dependency?

**Review by JackDiSalvatore — CHANGES_REQUESTED** [review]

**Review by HootDunk — CHANGES_REQUESTED** [review]
I went ahead and fixed these up in my PR so you can close this one

**Pull request closed without merging**